src/main/java/com/katsudon/binder/controller/ReadController.java

```diff
@@ -17,10 +19,15 @@
 public class ReadController {
   final CardRepository cardRepository;
   final ExpansionRepository expansionRepository;
+  final CardService cardService;
 
   @GetMapping("/cards")
-  public List<Card> findCards() {
-    return cardRepository.findAll();
+  public List<Card> findCards(@RequestParam List<String> rarity, @RequestParam List<String> set,
+                              @RequestParam String name, @RequestParam(defaultValue = "setReleaseDate") String orderBy,
+                              @RequestParam(defaultValue = "desc") String orderDir, @RequestParam(defaultValue = "1") int page,
+                              @RequestParam(defaultValue = "100") int pageSize) {
```

Comment:
I think the params would look better if it started on a new line, so it doesn't have the left padding.
Also, I recommend placing each param in a new line so its slightly easier to read.
I do like how you separated the optional params from the required ones

```java
  public List<Card> findCards(
    @RequestParam List<String> rarity,
    @RequestParam List<String> set,
    @RequestParam String name,
    @RequestParam(defaultValue = "setReleaseDate") String orderBy,
    @RequestParam(defaultValue = "desc") String orderDir,
    @RequestParam(defaultValue = "1") int page,
    @RequestParam(defaultValue = "100") int pageSize) {
  }
  ```